week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -33,7 +34,8 @@ console.log(y)
 ```
 
 What will be the output of this code. Explain your answer in 50 words or less.
-
+// on when function f1 call x value will be 10 as it is global variable and f1 function can access it
+//y will be undefined because console.log(y) have no acess to variable y without calling the function itself
```

Comment:
So what happens if you try and log or use a variable like y if it hasn't been declared in the global scope ?

Reply:
It will be undefined